app/jobs/replication/delivery_dispatcher_job.rb

```diff
@@ -26,15 +27,9 @@ class DeliveryDispatcherJob < Replication::ZipPartJobBase
     # @param [String] druid
     # @param [Integer] version
     # @param [String] part_s3_key, e.g. 'ab/123/cd/4567/ab123cd4567.v0001.z03'
-    # @param [Hash<Symbol => String>] metadata Zip info
-    # @option metadata [Integer] :parts_count
-    # @option metadata [Integer] :size
-    # @option metadata [String] :checksum_md5
-    # @option metadata [String] :suffix
-    # @option metadata [String] :zip_cmd
-    # @option metadata [String] :zip_version
```

Comment:
these are never used, and there is no `options` argument, so ... removing cruft.

Reply:
@ndushay Doesn't this unpack the `metadata` argument? Or is this better documented elsewhere, since all this job does with `metadata` is pass it on elsewhere?

Reply:
I may have misinterpreted the `@option` yard command -- as I have seen it in the past as @options.  I can leave in.

Reply:
indeed, I am in the wrong. https://rubydoc.info/gems/yard/file/docs/Tags.md#option.  Leaving in.